src/views/guestLayout.js

```diff
@@ -12,18 +12,18 @@ const guestLayout = data => `
 <!doctype html>
 <html lang=${getLocale()}>
   <head>
-    <title>${getMessage('brand-fx-monitor')}</title>
+    <title>${data.meta?.title ?? getMessage('brand-fx-monitor')}</title>
```

Comment:
nit: I wonder if it's better to do this above on L7.5 or something. Looks like we do lookups for `data.meta?.title ?? getMessage('brand-fx-monitor')` 1-3&times; (same w/ `socialDescription`).


```jsx
const metaTitle = data.meta?.title ?? getMessage('brand-fx-monitor');
const socialTitle = data.meta?.socialTitle ?? getMessage('brand-fx-monitor');
const socialDesc = data.meta?.socialDescription ?? getMessage('meta-desc-2');

...

<title>${metaTitle}</title>
<meta name='twitter:title' content='${socialTitle}' />
<meta name='twitter:description' content='${socialDesc}' />
<meta property='og:title' content='${socialTitle}' />
<meta property='og:description' content='${socialDesc}' />
```

Reply:
Good idea — done in: https://github.com/mozilla/blurts-server/pull/3009/commits/d39312fcf147b6c5c1d1d018b5776a42cdf101a8.